tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
```
'AA' => true,
'AB' => true,
'AC' => 'a',
```
minus
```
'AA' => false,
```
so 'AA' should be out as 'AA => false' is the way to turn it off (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2545#discussion_r107535379)?